powsybl-build-tools/src/main/resources/powsybl-build-tools/lombok.config

```diff
@@ -0,0 +1,14 @@
+# Don't search another lombok.config file in parent folders
+config.stopbubbling = true
+
+# We add annotations flags for IDEs and tools :
+#  * mark the code as generated for linters and coverage tools
+lombok.addJavaxGeneratedAnnotation = true
+lombok.addLombokGeneratedAnnotation = true
+#  * disable all warnings on generated code for checking tools and IDE (eg. checkstyle)
+lombok.addSuppressWarnings = true
+#  * we prefer the javax Nullable annotation for compatibility with majority of tools
+lombok.addNullAnnotations = javax
```

Comment:
"Many flavors are available: javax (=JSR305; not recommended), "
why do we use a not recommended conf ?

Reply:
remove ?

Reply:
> "Further more these annotations use javax. packages which it is not possible to use according to the Oracle Java binary licence [...]"
Oh, I haven't check that... I switch to this one at beginning because the majority of tools recognise javax annotations natively.  
Between the possibilities (~`javax`~, `jakarta`, `eclipse`, `jetbrains`, `netbeans`, ~`androidx`~, ~`android.support`~, `checkerframework` _(recommended)_, `findbugs`, `spring`, `jml`), I wanted to use `checkerframework`/`findbugs`/`spring`, but there aren't in dependencies by default ...